### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
I have a sample, let me commit that and then you can take this PR forward.

**Comment by smn** [comment]
@JayH5 @miltontony ready for review

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Mesos
+    http://localhost:5050
+
+Nginx
+    http://localhost:8000
+
+
+.. _Vagrant: http://www.vagrantup.com
+.. _Mesos: https://mesosphere.com/
```
Comment (JayH5) [inline README.rst]: Rather link to http://mesos.apache.org

**Comment by JayH5** [comment]
Not sure if worth mentioning consul-template?

**Comment by smn** [comment]
@JayH5 ready for re-review

**Comment by JayH5** [comment]
:+1:

**Pull request merged**